flytekit/core/python_function_task.py

```diff
@@ -13,11 +13,14 @@
 
 """
 
+from __future__ import annotations
+
 from abc import ABC
 from collections import OrderedDict
 from enum import Enum
-from typing import Any, Callable, List, Optional, TypeVar, Union, cast
+from typing import Any, Callable, Iterable, List, Optional, TypeVar, Union, cast
 
+from flytekit.core import launch_plan as _annotated_launch_plan
```

Comment:
you could use `TYPE_CHECKING` to avoid circular import
```suggestion
if TYPE_CHECKING:
    from flytekit.core import launch_plan as _annotated_launch_plan
```

Reply:
Thanks. I think https://github.com/flyteorg/flytekit/pull/2015/commits/6920d5bd1d791e990511a3f504a9b695cd835c1d also works and avoids the if statement.